From c02413504497dc1477fade9495ea26281d463b52 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 28 Apr 2014 14:42:42 +0200
Subject: [LIBREPORT PATCH 06/10] Bugzilla: move the advanced options to the
 advanced section

Keep only login and password fields in the configuration dialogue.

URL is not changed often enough to be the first field in the
configuration dialogue. Normal users do not changed neither URL option
nor SSL Verify option.

Private ticket option should also not been checked by default. The
reporting user might request to restrict access to a bug report during
the reporting process.

Related to #259

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/report_Bugzilla.xml.in | 34 +++++++++++++++++-----------------
 1 file changed, 17 insertions(+), 17 deletions(-)

diff --git a/src/plugins/report_Bugzilla.xml.in b/src/plugins/report_Bugzilla.xml.in
index 79f2bcd..7fc596e 100644
--- a/src/plugins/report_Bugzilla.xml.in
+++ b/src/plugins/report_Bugzilla.xml.in
@@ -12,34 +12,29 @@
     <gui-review-elements>yes</gui-review-elements>
 
     <options>
-        <option type="text" name="Bugzilla_BugzillaURL">
-            <_label>Bugzilla URL</_label>
-            <allow-empty>no</allow-empty>
-            <_description>Address of Bugzilla server</_description>
-            <default-value>https://bugzilla.redhat.com</default-value>
-            <_note-html>You can create bugzilla.redhat.com account &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</_note-html>
-        </option>
         <option type="text" name="Bugzilla_Login">
             <_label>User name</_label>
             <allow-empty>no</allow-empty>
             <_description>Bugzilla account user name</_description>
+            <_note-html>You can create bugzilla.redhat.com account &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</_note-html>
         </option>
         <option type="password" name="Bugzilla_Password">
             <_label>Password</_label>
             <allow-empty>no</allow-empty>
             <_description>Bugzilla account password</_description>
         </option>
-        <option type="bool" name="Bugzilla_SSLVerify">
-            <_label>Verify SSL</_label>
-            <_description>Check SSL key validity</_description>
-            <default-value>yes</default-value>
-        </option>
-        <option type="bool" name="Bugzilla_CreatePrivate">
-            <_label>Restrict access</_label>
-            <_description>Restrict access to the created bugzilla ticket allowing only users from specified groups to view it (see advanced settings for more details)</_description>
-            <default-value>no</default-value>
-        </option>
         <advanced-options>
+            <option type="text" name="Bugzilla_BugzillaURL">
+                <_label>Bugzilla URL</_label>
+                <allow-empty>no</allow-empty>
+                <_description>Address of Bugzilla server</_description>
+                <default-value>https://bugzilla.redhat.com</default-value>
+            </option>
+            <option type="bool" name="Bugzilla_SSLVerify">
+                <_label>Verify SSL</_label>
+                <_description>Check SSL key validity</_description>
+                <default-value>yes</default-value>
+            </option>
             <option type="text" name="Bugzilla_Product">
                 <_label>Bugzilla product</_label>
                 <allow-empty>yes</allow-empty>
@@ -60,6 +55,11 @@
                 <allow-empty>yes</allow-empty>
                 <_note-html>Sets the proxy server to use for HTTPS</_note-html>
             </option>
+            <option type="bool" name="Bugzilla_CreatePrivate">
+                <_label>Restrict access</_label>
+                <_description>Restrict access to the created bugzilla ticket allowing only users from specified groups to view it (see advanced settings for more details)</_description>
+                <default-value>no</default-value>
+            </option>
             <option type="text" name="Bugzilla_PrivateGroups">
                 <_label>Groups</_label>
                 <allow-empty>yes</allow-empty>
-- 
1.8.3.1

